apps/docs/src/.vitepress/components/IconLibraryItem.vue

```diff
@@ -0,0 +1,75 @@
+<script lang="ts" setup>
+import OnyxIcon from "~components/OnyxIcon/OnyxIcon.vue";
+import type { EnrichedIcon } from "../utils-icons";
+import { ref } from "vue";
+
+const props = defineProps<{
+  icon: EnrichedIcon;
+}>();
+
+const isCopied = ref(false);
+
+const handleCopy = async () => {
+  const { importName, iconName } = props.icon;
+  await navigator.clipboard.writeText(
+    `import ${importName} from "@sit-onyx/icons/${iconName}.svg?raw";`,
+  );
+  isCopied.value = true;
+  setTimeout(() => (isCopied.value = false), 3000);
+};
+</script>
+
+<template>
+  <div class="icon" tabindex="0" @click="handleCopy" @keyup.enter="handleCopy">
+    <OnyxIcon
+      :icon="props.icon.content"
+      :color="props.icon.metadata.deprecated ? 'secondary' : 'currentColor'"
```

Comment:
I know the color was not added in your PR but I don't think we should use secondary for deprecated because secondary is a colored hightlight color. Imho deprecated icons should be greyed out

Reply:
does neutral count as grey?
<img width="487" alt="image" src="https://github.com/SchwarzIT/onyx/assets/151019977/98d1ac83-8189-4731-b572-a81f38c5cc7b">
<img width="644" alt="image" src="https://github.com/SchwarzIT/onyx/assets/151019977/6aeaf63e-9dfc-4fd6-9369-d1e0722104fa">

Reply:
Neutral is the default icon color^^ 
Maybe we could add a class if the icon is deprecated an then add custom CSS to change the color?

Reply:
But you don't have to do this in this PR :)

Reply:
nope. 
<img width="731" alt="image" src="https://github.com/SchwarzIT/onyx/assets/151019977/9269f719-3b53-4969-aab2-2bb90369fc67">
attachment is for test reasons deprecated but neutral is the same as currentcolor in this case

Reply:
I'll ask jannick

Reply:
short response: deprecated is no use case for now. no special color treatment